chcemvediet/apps/__init__.py

```diff
@@ -35,7 +35,12 @@ def workaround_sqlite_format_dtdelta_bug(self):
 
             def new_sqlite_format_dtdelta(*args):
                 res = original_sqlite_format_dtdelta(*args)
-                return res.strip(u'+00:00') if res else None
+                if not res:
+                    return None
+                elif res.endswith(u'+00:00'):
+                    return res[:-6]
+                else:
+                    return res
```

Comment:
Treba tak kosato tie if/else? Nemohlo ostat `res[...] if res and res.endswith(...) else None`? Alebo ak mas pocit, ze je to vela v jednom riadku, tak len:
```
if res and res.endswith(...):
  return ...
else:
  return res
```

Reply:
:ok: ten inline nie je dobry, pretoze pre `dt`, ktory nema timezone by vratilo `None`.